cliquet_docs/api/timestamps.rst

```diff
@@ -64,9 +72,32 @@ In order to prevent race conditions, like overwriting changes occured in the int
 a ``If-Match`` request header can be used. If the response is ``412 Precondition failed``
 then the resource has changed meanwhile.
 
-The client can then choose to:
++-----------------------------+-----------------------------+-------------------+
+| **If-Match: "<timestamp>"** | Changed meanwhile           | Not changed       |
++=============================+=============================+===================+
+| POST {resource}             | ``412 Precondition failed`` | Create            |
++-----------------------------+-----------------------------+-------------------+
+| PUT {resource}/{id}         | ``412 Precondition failed`` | Create or replace |
++-----------------------------+-----------------------------+-------------------+
+| PATCH {resource}/{id}       | ``412 Precondition failed`` | Modify            |
++-----------------------------+-----------------------------+-------------------+
+| DELETE {resource}/{id}      | ``412 Precondition failed`` | Delete            |
++-----------------------------+-----------------------------+-------------------+
+
+Concurrency control also allows to make sure a creation won't overwrite any record:
+
++---------------------+-----------------------------+------------+
+| **If-None-Match: ***| Id exists                   | Id unknown |
++=====================+=============================+============+
+| POST {resource}     | ``412 Precondition failed`` | Create     |
++---------------------+-----------------------------+------------+
+| PUT {resource}/{id} | ``412 Precondition failed`` | Create     |
++---------------------+-----------------------------+------------+
```

Comment:
These tables would be a little more obvious (and helpful) to me if they featured the different operations with different results in a same table. Right now you have to compare two distinct tables in the page, which is possibly confusing to the reader.